packages/desktop-client/src/components/manager/WelcomeScreen.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import * as actions from 'loot-core/src/client/actions';
+import {
+  View,
+  Button,
+  Text,
+  P,
+  ExternalLink,
+} from 'loot-design/src/components/common';
+import { colors, styles } from 'loot-design/src/style';
+
+function WelcomeScreen({ createBudget, pushModal }) {
+  return (
+    <View
+      style={{
+        gap: 10,
+        maxWidth: 500,
+        fontSize: 15,
+        maxHeight: '100vh',
+        marginBlock: 20,
+      }}
+    >
+      <Text style={styles.veryLargeText}>Let’s get started!</Text>
+      <View
+        style={{
+          overflowY: 'auto',
+        }}
+      >
+        <P>
+          Actual is a personal finance tool that focuses on beautiful design and
+          a slick user experience.{' '}
+          <strong>Editing your data should be as fast as possible.</strong> On
+          top of that, we want to provide powerful tools to allow you to do
+          whatever you want with your data.
+        </P>
+        <P>
+          Currently Actual implements budgeting based on a{' '}
+          <ExternalLink
+            asAnchor
+            style={{ color: colors.p5 }}
+            href="https://actualbudget.github.io/docs/Budgeting/howitworks"
+          >
+            monthly envelope system
+          </ExternalLink>
+          . In the future, we’ll support multiple ways to do budgeting. We’re
+          also working hard on custom reports and a lot more things.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I don't think we should promise future features. Look at the reports page: it has had "more reports coming soon" banner for like 1? maybe 2 years now?
```suggestion
          .
```

Reply:
3, maybe 4 :)

This was copied from the old modal but I agree it should be removed.